bin/transform.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+import argparse
+import pathlib
+import pandas
+import numpy
+
+from dateutil.relativedelta import relativedelta
+from datetime import datetime
+
+# Column headers:
+SAMPLE_HEADER = "sample"
+SAMPLE_NAME_HEADER = "sample_name"
+AGE_HEADER = "age"
+VALID_HEADER_EXTENSION = "_valid"
+ERROR_HEADER_EXTENSION = "_error"
+
+# Column indices:
+DATE_1_INDEX = 2
+DATE_2_INDEX = 3
+
+# Transformations:
+LOCK = "lock"
+AGE = "age"
+
+# Other:
+AGE_THRESHOLD = 2 # Ages less than this will include a decimal component.
+DAYS_IN_YEAR = 365.0
+
+def remove_empty_columns(metadata):
+    metadata.dropna(axis="columns", how="all", inplace=True)
+
+def lock(metadata):
+    metadata_readable = metadata.copy(deep=True)
+    metadata_irida = metadata.drop(columns=[SAMPLE_NAME_HEADER], inplace=False)
+
+    return metadata_readable, metadata_irida
+
+def format_age(age):
+    if age < AGE_THRESHOLD:
+        formatted_age = "{:.4f}".format(age)
+    else:
+        formatted_age = "{:.0f}".format(age)
```

Comment:
#I forget the exact meaning of float formatting, but does this round to nearest integer, or is just dropping decimal values? I think it rounds but could you confirm?

Could you also add a node in the readme that ages are rounded.

Reply:
Just confirming, do you want me to check if there's rounding for the 4-decimal place number? Whether or not `1.23456` becomes `1.2345` or `1.2346`? I'm almost certain this has no effect on the "integer" ages (they're actually integers converted to floats since the column is float), but I'll double check.

Reply:
Check for both 4-decimal numbers and integers.

I wrote the comment prior to seeing the other bit of code for calculating age >= 2 years. So this comment wouldn't really apply there right now (since they're already integers).

Reply:
Thanks so much Eric :)